Makefile.js

```diff
@@ -708,6 +736,51 @@ target.checkLicenses = function() {
     });
 };
 
+target.checkGitCommit = function() {
+    var commitMsgs,
+        failed;
+
+    if (hasBranch("master")) {
```

Comment:
Have a look at the Travis environment variables
http://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables
Maybe you can get all needed information from there. You could also use the
github api instead of the git commands.
On Jul 20, 2015 6:11 PM, "Ian VanSchooten" <notifications@github.com> wrote:

> In Makefile.js
> <https://github.com/eslint/eslint/pull/3059#discussion_r35012217>:
>
> > @@ -708,6 +736,51 @@ target.checkLicenses = function() {
> >      });
> >  };
> >
> > +target.checkGitCommit = function() {
> > +    var commitMsgs,
> > +        failed;
> > +
> > +    if (hasBranch("master")) {
>
> Sorry for the flurry here, but I checked a non-commiter PR Travis build
> <https://travis-ci.org/eslint/eslint/jobs/71743413>, and unless I'm
> reading it wrong, it appears to indeed pull in master from eslint/eslint.
>
> Not sure if there's really a good way to test this, unless you merge, and
> then revert if builds start breaking incorrectly.
>
> I'm open to suggestions how to proceed here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eslint/eslint/pull/3059/files#r35012217>.
>